test/colony-network.js

```diff
@@ -129,21 +129,18 @@ contract("Colony Network", accounts => {
     it("when meta colony is created, should have the root global and local skills initialised, plus the local mining skill", async () => {
       const skillCount = await colonyNetwork.getSkillCount();
       expect(skillCount).to.eq.BN(3);
-      const rootGlobalSkill = await colonyNetwork.getSkill(1);
-      expect(parseInt(rootGlobalSkill.nParents, 10)).to.be.zero;
-      expect(parseInt(rootGlobalSkill.nChildren, 10)).to.be.zero;
+      const globalSkill = await colonyNetwork.getSkill(3);
```

Comment:
I feel like adding a `ROOT_GLOBAL_SKILL = 3` to `constants.js` would be a good idea vs. having that as implicit knowledge.